apps/nextjs-website/src/components/atoms/CtaCard/CtaCard.tsx

```diff
@@ -22,6 +24,7 @@ export type CtaCardProps = {
   readonly comingSoon?: boolean;
   readonly icon?: ReactNode;
   readonly children?: ReactNode | ReactNode[];
+  readonly tags?: { readonly label: string; readonly path?: string }[];
```

Comment:
This is not an array of `Products`, you should change the changeset file
`Refactor CtaCard to allow an array of tags in input`